### PR TITLE
Feature/python sdk exception handling

### PR DESCRIPTION
**Pull request by eduwercamacaro**

### CODE REVIEW

File: sdk-python/examples/error_handling/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## Running with an External Event
```
Comment (coltmcnealy-lh) [inline sdk-python/examples/error_handling/README.md:1]: Title looks a little off...

File: sdk-python/examples/error_handling/example_error_handling.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        config.load(config_path)
+    return config
+
+
+def entrypoint(thread: ThreadBuilder) -> None:
+    def exception_handler(thread: ThreadBuilder) -> None:
+        thread.execute("handler")
+
+    node = thread.execute("fail")
+    thread.handle_error(node, exception_handler, LHErrorType.TASK_ERROR)
```
Comment (coltmcnealy-lh) [inline sdk-python/examples/error_handling/example_error_handling.py:27]: Nit pick: in the other SDK's, the order of the arguments is:
1. `NodeOutput`
2. Error type
3. Error handler function

File: sdk-python/tests/test_workflow.py [hunk 3]
```diff
@@ -795,6 +796,285 @@ def my_entrypoint(thread: ThreadBuilder) -> None:
...
+
+        def my_entrypoint(thread: ThreadBuilder) -> None:
+            node = thread.execute("fail")
+            thread.handle_any_failure(node, my_interrupt_handler)
+            thread.execute("my-task")
+
+        wf = Workflow("my-wf", my_entrypoint)
+        self.assertEqual(
+            wf.compile(),
+            PutWfSpecRequest(
```
Comment (coltmcnealy-lh) [inline sdk-python/tests/test_workflow.py:811]: This works! You just had to out-do my tests in Go :laughing: I just verified specific node's. Well done

**Review by coltmcnealy-lh — APPROVED** [review]

File: sdk-python/examples/error_handling/example_error_handling.py (hunk 2 above)
Comment (sauljabin) [reply sdk-python/examples/error_handling/example_error_handling.py:27]: let me check

Comment (sauljabin) [reply sdk-python/examples/error_handling/example_error_handling.py:27]: that is true but now error type defaults None, so it has to be at the end

**Review by sauljabin — APPROVED** [review]

**Pull request merged**